### ISSUE TITLE
On Update: open update page with changelogs

### ISSUE THREAD

**Issue by GODrums** [GODrums/BetterFloat#13]

**Comment by bycop** [comment]
For this, you just have to create an html, url or anything from the onInstalled listener by example
`chrome.tabs.create({ url: 'html/changelog.html' });`

**Comment by GODrums** [comment]
Yeah, from a technical standpoint, this is quite easy, but just reiterating the changelog brings no additional value. And I feel like it would be very annoying to get a popout page every few days.

I'm looking for a more subtle solution, which can also provide additional value via pictures or more detailed explanations of how features work / how to use them.
